zebra-chain/src/orchard/tests/vectors.rs

```diff
@@ -0,0 +1,47 @@
+use halo2::arithmetic::FieldExt;
+use halo2::pasta::pallas;
+
+use crate::orchard::tests::test_vectors;
+use crate::orchard::tree::*;
+
+#[test]
+fn empty_roots() {
+    zebra_test::init();
+
+    for i in 0..EMPTY_ROOTS.len() {
+        assert_eq!(
+            EMPTY_ROOTS[i].to_bytes(),
+            // The test vector is in reversed order.
+            test_vectors::EMPTY_ROOTS[MERKLE_DEPTH - i]
+        );
+    }
+}
+
+#[test]
+fn incremental_roots() {
+    zebra_test::init();
+
+    let mut leaves = vec![];
+
+    let mut incremental_tree = NoteCommitmentTree::default();
+
+    for (i, commitment_set) in test_vectors::COMMITMENTS.iter().enumerate() {
```

Comment:
Since i is never used, you could skip the `enumerate`

Reply:
It's used below to get the matching root for this set of commitments